docs/backend/controlpanels.md

```diff
@@ -0,0 +1,43 @@
+---
+html_meta:
+  "description": ""
+  "property=og:description": ""
+  "property=og:title": ""
+  "keywords": ""
+---
+
+(backend-controlpanels-label)=
+
+# Controlpanels
+
+## Adding a controlpanel
+To add a controlpanel to your add-on you can use [`plonecli`](https://pypi.org/project/plonecli/) follows:
+
+```shell
+plonecli add controlpanel
+```
+This will create the controlpanel python file in the controlpanel's folder where you can define your controlpanel schema fields. 
```

Comment:
```suggestion
This will create the control panel Python file in the control panel's folder where you can define your control panel schema fields. 
```